packages/opentelemetry-api/src/trace/spancontext-utils.ts

```diff
@@ -13,24 +13,34 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { SpanContext } from './span_context';
+import { TraceFlags } from './trace_flags';
 
-import { SpanContext, TraceFlags } from '@opentelemetry/api';
-
-export const INVALID_SPANID = '0';
-export const INVALID_TRACEID = '0';
+const VALID_TRACEID_REGEX = /^([0-9a-f]{16}){1,2}$/i;
```

Comment:
```suggestion
const VALID_TRACEID_REGEX = /^([0-9a-f]{32})$/i;
```

Reply:
Hmm, I guess I should use a different const for the b3 propagator since it can be either 16 or 32 characters long on extract, but then should be padded with 0's